include/roboteam_ai/bt/composites/Selector.hpp

```diff
@@ -6,6 +6,7 @@ namespace bt {
 
 class Selector : public Composite {
 public:
+    Selector();
```

Comment:
Why add all these empty constructors? (Also goes for all other composite.h files) Isn't it better to initialize explicitly so certain nodes MUST have children?

Reply:
Because std::make_unique<selector> requires a default constructor, we construct a map somewhere of it std::string to std::shared_ptr<Node> iirc, which requires a default ctor, stl things.